.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+on: [push, pull_request]
+env:
+  CI: true
+  SCALA_VERSION: 2.13.1
+jobs:
+  validate:
+    name: Scala ${{ matrix.scala }}, Java ${{ matrix.java }}
+    runs-on: ${{matrix.systems}} 
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [adopt@1.8, adopt@1.11, adopt@1.14,graalvm@]
+        scala: [2.12.11, 2.13.1]
+        #TODO: windows-latest likely requires another [job](https://github.com/olafurpg/setup-scala)
+        systems: [ubuntu-latest, macos-latest, windows-latest]
+    env:
+      SCALA_VERSION: ${{ matrix.scala }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
+        with:
+          java-version: ${{ matrix.java }}
+      # TODO: should we remove?
+      - name: Cache Coursier
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/coursier
+          key: sbt-coursier-cache
+      # TODO: should we remove?     
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: sbt-${{ hashFiles('**/build.sbt') }}
+      - name: g8Test
+        if:  startsWith(matrix.scala,'2.12.11')
+        run: sbt ++$SCALA_VERSION g8Test 
```

Comment:
Any reason that this doesn't work for the whole matrix, without the `if`?  And then eliminate the next step?

Reply:
Yes, it fails in the CI because sbt doesn't have you set the version when you are using that version. E.G. if you are using scala 2.13.1 and run sbt ++2.13.1 g8Test it will error. You will see it in the commit messages and a failed build.

Reply:
Oh, in 49753d9?  I don't see failed builds, but for whatever reason, your GitHub Actions runs aren't showing up on this pull request.  I am not yet sure why they're only visible on your fork.

Reply:
Maybe because you aren't a contributor on my fork?

Reply:
I think the way it works you have to have the .github folder merged to execute the CI task. Then you won't see a success or failure against the http4s repository until a merge happens. I can understand that as a deficiency of the platform. I am not sure if there's another option that will put the success or failure of my actions forward into this PR.

Reply:
Ah, I'll bet you're right.  I think every time I've set one of these up, I started with a PR from the target repo, instead of from a fork.  I can see them running from the Actions view on your fork.  Let's just assume they'll show up here once it's merged, and revisit if they don't.